server/src/test/java/org/opensearch/cluster/coordination/JoinTaskExecutorTests.java

```diff
@@ -183,7 +183,13 @@ public void testUpdatesNodeWithNewRoles() throws Exception {
             actualNode.getVersion()
         );
         final ClusterState clusterState = ClusterState.builder(ClusterName.DEFAULT)
-            .nodes(DiscoveryNodes.builder().add(masterNode).localNodeId(masterNode.getId()).masterNodeId(masterNode.getId()).add(bwcNode))
+            .nodes(
+                DiscoveryNodes.builder()
+                    .add(clusterManagerNode)
+                    .localNodeId(clusterManagerNode.getId())
+                    .masterNodeId(clusterManagerNode.getId())
```

Comment:
masterNodeId -> clusterManagerNodeId?

Reply:
I will rename `masterNodeId()` in the next phase, which is mentioned in issue https://github.com/opensearch-project/OpenSearch/issues/1684.